vendor/ocaml-4.13-extended/parser.mly

```diff
@@ -1094,8 +1094,8 @@ reversed_bar_llist(X):
 listx(delimiter, X, Y):
 | x = X ioption(delimiter)
     { [x], None }
-| x = X delimiter y = Y delimiter?
-    { [x], Some y }
+| x = X delimiter Y delimiter?
+    { [x], Some (make_loc $loc($3)) }
```

Comment:
We can keep `y` and reduce the diff:
```suggestion
| x = X delimiter y = Y delimiter?
    { ignore y; [x], Some (make_loc $loc(y)) }
```

Reply:
I chose a slightly different fix to reduce the diff:
```diff
- | x = X delimiter y = Y delimiter?
+ | x = X delimiter y = mkloc(Y) delimiter?
    { [x], Some y }
```

Reply:
Nice :)